webhooks.md

```diff
@@ -0,0 +1,114 @@
+# Webhooks
+
+Whenever integration client is changing its state webhook message is sent to certain webhook url field set on that client. Webhook url is instatiated per integration and responds on action made on property level.
```

Comment:
I would write something like Action is triggered everytime when hotel update the integration or smth like that.
Or any change on integration triggers the webhook...

Reply:
Action is triggered everytime...